docs/validator/running-a-validator.md

```diff
@@ -0,0 +1,252 @@
+
+# Start a New Gno Chain and a Validator
+
+## 1. Initialize the configurations (required)
+
+```bash
+gnoland config init
+```
+
+## 2. Initialize the secrets (required)
+
+```bash
+gnoland secrets init
+```
+
+## 3. Set a moniker (optional)
+
+```bash
+gnoland config set moniker node01
+```
+
+## 4. Set the rpc connection address (required for connecting with other nodes)
+
+```bash
+gnoland config set rpc.laddr "tcp://0.0.0.0:26657"
+
+# similar behavior for cosmos validator
+# gaiad tx staking create-validator `--node string (default:tcp://localhost:26657)`
+```
+
+## 5. Set the validator private key (required)
+
+:::tip
+
+The key file path is relative by default.
+
+:::
+
+```bash
+gnoland config set priv_validator_key_file secrets/priv_validator_key.json
+```
+
+## 6. Set the validator state (required)
+
+:::tip
+
+The key file path is relative by default.
+
+:::
+
+```bash
+gnoland config set priv_validator_state_file secrets/priv_validator_state.json
+```
+
+## 7. Set the node key (required)
+
+:::tip
+
+The key file path is relative by default.
+
+:::
+
+```bash
+gnoland config set node_key_file secrets/node_key.json
+```
+
+## 8. Set the genesis file (required)
+
+```bash
+gnoland genesis generate
+```
+
+## 9. Add a validator (required)
+
+```bash
+# check the secrets file generated in step (2)
+$ gnoland secrets get
+[Node P2P Info]
+Node ID:  g19d8x6tcr2eyup9e2zwp9ydprm98l76gp66tmd6
+
+[Validator Key Info]
+Address:     g1lnha5yem9dmj0yfzysfqsnvrm6j2ywshq83qdf
+Public Key:  gpub1pggj7ard9eg82cjtv4u52epjx56nzwgjyg9zpleysamt23ar025757uepld60xztnw7ujc3gwtjuy4pwv6z9sh4g284h3q
+
+[Last Validator Sign State Info]
+Height:  0
+Round:   0
+Step:    0
+
+# add the validator to the genesis file using the address and the public key in the Validator Key Info section
+$ gnoland genesis validator add -address g1lnha5yem9dmj0yfzysfqsnvrm6j2ywshq83qdf -name node01 -power 10 -pub-key gpub1pggj7ard9eg82cjtv4u52epjx56nzwgjyg9zpleysamt23ar025757uepld60xztnw7ujc3gwtjuy4pwv6z9sh4g284h3q
```

Comment:
This command should be split in newlines to lay out the different flags, and document them

Reply:
5736d11b092bdbc038c5c45a99db48cf0bfef785